recipes/click-loglevel/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
`LICENSE` is already included in the PyPI tarball, so you shouldn't include it in the recipe.

Reply:
Removed.